ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotValueStyleFormat.cs

```diff
@@ -1,30 +1,40 @@
 ﻿// Keep this file CodeMaid organised and cleaned
 using System;
+using System.Collections.Generic;
 
 namespace ClosedXML.Excel
 {
-    internal class XLPivotValueStyleFormat : XLPivotStyleFormat, IXLPivotValueStyleFormat
+    internal class XLPivotValueStyleFormat : IXLPivotValueStyleFormat
     {
-        public XLPivotValueStyleFormat(IXLPivotField field = null, IXLStyle style = null)
-            : base(field, style)
-        { }
+        public XLPivotValueStyleFormat(IXLPivotFormat pivotFormat)
+        {
+            PivotFormat = pivotFormat;
+        }
 
         #region IXLPivotValueStyleFormat members
 
+        public IXLStyle Style
+        {
+            get => PivotFormat.Style;
+            set => PivotFormat.Style = value;
+        }
+
+        public IXLPivotFormat PivotFormat { get; }
+
         public IXLPivotValueStyleFormat AndWith(IXLPivotField field)
         {
             return AndWith(field, null);
         }
 
         public IXLPivotValueStyleFormat AndWith(IXLPivotField field, Predicate<Object> predicate)
         {
-            FieldReferences.Add(new PivotLabelFieldReference(field, predicate));
+            ((List<IFieldRef>)PivotFormat.FieldReferences).Add(FieldRef.ForField(field.SourceName, predicate));
```

Comment:
Better avoid typecast here. For example, in the constructor, you can accept a concrete implementation of `IXLPivotFormat` that allows modification of `FieldReferences` collection. Or define an internal interface for that. I can provide a more detailed explanation if needed.